python/understack-workflows/understack_workflows/main/print_bmc_password.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+
+from understack_workflows.bmc_password_standard import standard_password
+
+
+def main(program_name, bmc_ip_address=None):
+    """CLI script to obtain standard BMC Password.
+
+    Requires the master secret to be available in BMC_MASTER environment
+    variable.
+    """
+    if bmc_ip_address is None:
+        print(f"Usage: {program_name} <BMC IP Address>", file=sys.stderr)
+        exit(1)
+
+    if os.getenv("BMC_MASTER") is None:
+        print("Please set the BMC_MASTER environment variable")
+        exit(1)
+
+    password = standard_password(bmc_ip_address, os.getenv("BMC_MASTER"))
+    print(password)
+
+
+main(*sys.argv)
```

Comment:
Where's this file intended to be called from?

Reply:
I don't know - I put this here as a way for us undercloud admins to get the password for a given BMC.   I was thinking there must be some kubernetes way to run a script with access to the relevant secrets, but I have never seen anything like that.  If all else fails I will make an entire web app specifically to expose this functionality.